app/src/main/java/com/codesoom/assignment/handlers/TaskHttpHandler.java

```diff
@@ -0,0 +1,173 @@
+package com.codesoom.assignment.handlers;
+
+import com.codesoom.assignment.http.HttpMethod;
+import com.codesoom.assignment.http.HttpStatus;
+import com.codesoom.assignment.models.ResponseDto;
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.models.TaskDto;
+import com.codesoom.assignment.services.TaskService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.Headers;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class TaskHttpHandler implements HttpHandler {
+
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+    private final TaskService taskService;
+
+    private static final String PATH = "/tasks";
+    private static final String PATH_REGEX = "^\\/tasks\\/[0-9]+$";
```

Comment:
`/`는 이스케이프하지 않아도 작동할 것 같군요.
```suggestion
    private static final String PATH_REGEX = "^/tasks/[0-9]+$";
```